tests/ci/integration/run_accp_integration.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+set -exu
+
+source tests/ci/common_posix_setup.sh
+
+# Assumes script is executed from the root of aws-lc directory
+SCRATCH_FOLDER=${SRC_ROOT}/"scratch"
+ACCP_SRC="${SCRATCH_FOLDER}/amazon-corretto-crypto-provider"
+
+function build_and_test_accp() {
+  pushd "${ACCP_SRC}"
+  export JAVA_HOME=$(readlink -f /usr/bin/javac | sed "s:/bin/javac::")
+  # Assign the JDK version we're testing as the system's default JDK and
+  # assign JAVA_HOME variable to the path. Otherwise, Ubuntu will
+  # default to the newest version of Java on the system.
```

Comment:
Nit: Seems like we are defaulting to the newest JDK version on the runner here. No issue with that, but it does make this comment obsolete.

Reply:
Updated!